### PR TITLE
Phone numbers file

### PR DESCRIPTION
**Pull request by JoseHernandezRetana**
### que implementaciones o arreglos se le  hicieron en este PR.
this PR adds the txt that haves phone numbers
este 

### Details to clarify more about this PR:
When you need more phone numbers make me know about it before any change.
 
### What areas or files were affected by this change:
in this case i only add a new file to the project called DatosTelefonos.txt

### Where should the reviewers start to review this PR:
every one should start with the DatosTelefonos.txt

### How should this PR be tested:
Put the new txt called on the new data base that is not added yet

### Screenshots or videos:
no screenshots available Very well

### CODE REVIEW

File: DatosTelefonos.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+614 123467890
```
Comment (maztermarcoz) [inline DatosTelefonos.txt:1]: Este es mi numero Infeliz

File: DatosTelefonos.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+614 223344688
+614 436577618
+614 437458292
+614 701970951
```
Comment (AngelHdezRetana) [inline DatosTelefonos.txt:5]: Este numero no deberia de estar en la lista, porfa has ese cambio

Comment (JoseHernandezRetana) [reply DatosTelefonos.txt:5]: ok mister pancho

**Review by AngelHdezRetana — APPROVED** [review]

**Review by maztermarcoz — APPROVED** [review]
Ya veo que te la rifaste, todo excelente

**Pull request merged**